pyroengine/engine.py

```diff
@@ -242,7 +242,7 @@ def predict(self, frame: Image.Image, cam_id: Optional[str] = None) -> float:
             to_be_staged = self._update_states(pred, cam_key)
             if to_be_staged and len(self.api_client) > 0 and isinstance(cam_id, str):
                 # Save the alert in cache to avoid connection issues
-                self._stage_alert(frame, cam_id)
+                self._stage_alert(frame_resize, cam_id)
```

Comment:
If `self.frame_size` is not set, I think this is gonna crash :sweat_smile:

Reply:
good catch I will fix it !